save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/FileUtils.kt

```diff
@@ -56,6 +58,17 @@ fun Flux<DataBuffer>.writeTo(target: Path): Mono<Path> =
             .map { DataBufferUtils.release(it) }
             .then(Mono.just(target))
 
+/**
+ * @param target path to file to where a content from receiver will be written
+ * @return [Mono] with [target]
+ */
+fun Flux<ByteBuffer>.writeToFile(target: Path): Mono<Path> = map { byteBuffer ->
```

Comment:
I see you added this method, but I don't see that it was removed from the old place (AbstractFileBasedStorage). Can we reuse a common place of this method

Reply:
Will do it

Reply:
Done